invenio_app_ils/ui/backoffice/src/pages/UserDetails/components/PatronLoansTable/state/selectors.js

```diff
@@ -0,0 +1,19 @@
+import _isEmpty from 'lodash/isEmpty';
```

Comment:
Might be better to have this reusable to avoid duplication? As response serialization for any loan maybe?